### PR TITLE
Added documentation to help future DS student to deploy to EB AWS

### PR DESCRIPTION
**Pull request by jbehringer95**
# Description

Added detail documentation to help future DS students to deploy to EB AWS

## Type of change

- [x] This change requires a documentation update

## Change status

- [x] Complete, tested, ready to review and merge

# Checklist:
Everything in this checklist is required for each PR.  Please do not approve a PR that does not have all of these items.  

- [x] I have pulled from master before submitting this PR
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer (the peer review to approve a PR counts.  The reviewer must download and test the code)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by Jack4589** [comment]
Looks good!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## How to Use the DS API
 
-Main URL: http://family-promises-db.eba-saefv7mf.us-east-1.elasticbeanstalk.com
+Main URL: http://family-promise-dev.us-east-1.elasticbeanstalk.com
```
Comment (santorap) [inline README.md:3]: Much better URL

File: README.md [hunk 2]
```diff
@@ -67,7 +67,23 @@ database.
 
 - Create the .env file in the folder.
   - Add `DATABASE_URL = postgresql://docker:****@localhost:5400/api-dev` to the .env file.
+    - That is only for spinning up the local instance for the DS API
```
Comment (santorap) [inline README.md:70]: May want to make a comment specifying that the password needs to be manually added too.

**Review by santorap — APPROVED** [review]
Great job on updating the documentation.

**Review by Jack4589 — APPROVED** [review]

**Pull request merged**